hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/stream/DirstreamClientHandler.java

```diff
@@ -80,6 +80,10 @@ public void doRead(ChannelHandlerContext ctx, ByteBuf buffer)
         name.release();
         buffer.skipBytes(1);
         String[] parts = currentFileName.split(" ", 2);
+        if (parts.length < 2 || parts[1].isEmpty()) {
+          throw new IllegalArgumentException("Invalid file name format: " + currentFileName
+              + ". Expected format: SIZE FILENAME");
```

Comment:
Could you improve the message here to make it clear?
Something like this -

```suggestion
          throw new IllegalArgumentException("Invalid file name format: " + currentFileName + ". " +
              "Expected format: <size> <filename> where <size> is a number and <filename> is a string " +
              "separated by a single space. Example: '1024 myfile.txt'");
```